modelstore/metadata/storage/storage.py

```diff
@@ -30,8 +30,9 @@ class Storage:
     type: str
 
     # Path-like storage (e.g. local)
+    root: Optional[str] = field(default=None, metadata=config(exclude=exclude_field))
     path: Optional[str] = field(default=None, metadata=config(exclude=exclude_field))
```

Comment:
New field in the meta data for local storage, splitting out the root from the path